robustirc-bridge/bridge.go

```diff
@@ -41,6 +48,41 @@ var (
 // - for resuming sessions (later): the last seen message id, perhaps setup messages (JOINs, MODEs, …)
 // for hosted mode, this state is stored per-nickname, ideally encrypted with password
 
+func prefixMotd(msg string) string {
+	// The user chose to not inject a MOTD.
+	if *motdPath == "" {
+		return msg
+	}
+
+	sep := strings.Index(msg[1:], ":")
+	if sep == -1 {
+		return msg
+	}
+
+	prefix := msg[:sep+2] + "- "
```

Comment:
I have absolutely no clue what is happening here, just reading the code. Maybe add an example or something in a comment that makes somewhat clear, what is happening, to a person who hasn't read the IRC RFC.